pkgs/applications/science/logic/cadical/default.nix

```diff
@@ -11,14 +11,28 @@ stdenv.mkDerivation rec {
     sha256 = "05lvnvapjawgkky38xknb9lgaliiwan4kggmb9yggl4ifpjrh8qf";
   };
 
+  outputs = [ "out" "dev" "lib" ];
   doCheck = true;
-  dontAddPrefix = true;
+
+  configurePhase = ''
+    runHook preConfigure
+
+    ./configure
+
+    runHook postConfigure
+  '';
```

Comment:
Isn't this the default?

Reply:
no, the default phase adds many flags like `--prefix` `--libdir` etc. that are only understood by autoconf-generated configure scripts.

Reply:
Please add a comment that this is not an autoconf configure script.

Reply:
done